kube_apiserver_metrics/datadog_checks/kube_apiserver_metrics/kube_apiserver_metrics.py

```diff
@@ -75,8 +75,14 @@ def _create_kube_apiserver_metrics_instance(self, instance):
         """
         kube_apiserver_metrics_instance = deepcopy(instance)
         endpoint = instance.get('prometheus_url')
-        scheme = instance.get('scheme', self.DEFAULT_SCHEME)
-        kube_apiserver_metrics_instance['prometheus_url'] = "{0}://{1}".format(scheme, endpoint)
+        prometheus_url = endpoint
+
+        # Allow using a proper URL without introducing a breaking change.
+        if not match('^https?://.*$', endpoint):
+            scheme = instance.get('scheme', self.DEFAULT_SCHEME)
```

Comment:
(nit) If I understand correctly, we are deprecating `scheme` config. If so, do you mind adding a comment about `scheme` deprecation ?

Reply:
Good catch, thanks